src/uv-common.h

```diff
@@ -323,4 +323,13 @@ void* uv__malloc(size_t size);
 void uv__free(void* ptr);
 void* uv__realloc(void* ptr, size_t size);
 
+typedef struct {
```

Comment:
Nitpick, but I think we declare similar structures using `struct uv_metrics_s {...}` and `typedef struct uv_metrics_s uv_metrics_t`

Reply:
I knew this was the case in `include/uv.h` but wasn't sure about the code in `src/`. So I took my example from `uv__allocator_t` in `src/uv-common.cc`. But after doing a quick `grep` I see that pattern is still usually the case here. I'll change it to follow that pattern.